apps/src/StudioApp.js

```diff
@@ -589,25 +589,29 @@ StudioApp.prototype.init = function (config) {
   }
 };
 
+StudioApp.prototype.getVersionHistoryHandler = function (config) {
+  return () => {
+    var codeDiv = document.createElement('div');
```

Comment:
While you're extracting this, `codeDiv` seems like a misnomer, maybe a leftover copy-paste fragment.  How about just calling it `contentDiv`?